src/libraries/System.Private.CoreLib/src/System/Threading/PortableThreadPoolEventSource.cs

```diff
@@ -3,117 +3,286 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Diagnostics.Tracing;
+using System.Runtime.CompilerServices;
+using Internal.Runtime.CompilerServices;
 
 namespace System.Threading
 {
-    [EventSource(Name = "Microsoft-Windows-DotNETRuntime", Guid = "{e13c0d23-ccbc-4e12-931b-d9cc2eee27e4}")]
-    public sealed class PortableThreadPoolEventSource : EventSource
+    [EventSource(Name = "Microsoft-Windows-DotNETRuntime", Guid = "e13c0d23-ccbc-4e12-931b-d9cc2eee27e4")]
+    internal sealed class PortableThreadPoolEventSource : EventSource
     {
-        private const string WorkerThreadMessage = "WorkerThreadCount=%1";
-        private const string WorkerThreadAdjustmentSampleMessage = "Throughput=%1";
-        private const string WorkerThreadAdjustmentAdjustmentEventMessage = "AverageThroughput=%1;%nNewWorkerThreadCount=%2;%nReason=%3";
-        private const string WorkerThreadAdjustmentStatsEventMessage = "Duration=%1;%nThroughput=%2;%nThreadWave=%3;%nThroughputWave=%4;%nThroughputErrorEstimate=%5;%nAverageThroughputErrorEstimate=%6;%nThroughputRatio=%7;%nConfidence=%8;%nNewControlSetting=%9;%nNewThreadWaveMagnitude=%10";
+        // This value does not seem to be used, leaving it as zero for now. It may be useful for a scenario that may involve
+        // multiple instances of the runtime within the same process, but then it seems unlikely that both instances' thread
+        // pools would be in moderate use.
+        private const short ClrInstanceId = 0;
```

Comment:
Yup, this is a historical artifact.  Seems fine to leave it if you're trying to match the exact arguments as the native versions of these events.  Otherwise, you could get rid of this argument.